test/client/components/calls/call_question.test.js

```diff
@@ -24,39 +30,28 @@ const props2 = {
   change: jest.fn(),
   form: 'Forms'
 }
+const props3 = {
+  question: paragraph,
+  change: jest.fn(),
+  form: 'Forms'
+};
 describe('<CallQuestion />', () => {
   const wrapper1 = shallow(<CallQuestion {...props1} />);
   const wrapper2 = shallow(<CallQuestion {...props2} />);
-  describe('rendering general', () => {
-    it('should render a description in <ControlLabel>', () => {
-      expect(wrapper1.find('ControlLabel').first().html()).toContain(props1.question.description);
-    });
-    it('should render a <Badge>', () => {
-      expect(wrapper1.find('Badge').length).toBe(1);
```

Comment:
Removed these tests because this rendering is now happening in the common components `RadioGroup` and `CheckboxGroup`